src/main/java/com/google/sps/servlets/GetPostServlet.java

```diff
@@ -0,0 +1,60 @@
+package com.google.sps.servlets;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.Query;
+import com.google.cloud.datastore.QueryResults;
+import com.google.cloud.datastore.StructuredQuery.OrderBy;
+import com.google.gson.Gson;
+import com.google.sps.data.Task;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible for loading posts. */
+@WebServlet("/posts")
+public class GetPostServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
+    Query<Entity> query =
+        Query.newEntityQueryBuilder().setKind("Post").setOrderBy(OrderBy.desc("timePosted")).build();
+    QueryResults<Entity> results = datastore.run(query);
+
+    List<Post> posts = new ArrayList<>();
+    while (results.hasNext()) {
+      Entity entity = results.next();
+
+      long id = entity.getKey().getId();
```

Comment:
Please make sure to use the right names for the database variables, they should be the same throughout the Post.Java class and the database